src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2573,7 +2573,9 @@ public static void GetSystemTimeZones_AllTimeZonesHaveOffsetInValidRange()
             // 0x3E, 0x2C, 0x30, 0x2F, 0x30, 0x2C, 0x4A, 0x33, 0x36, 0x35, 0x2F, 0x32, 0x35, 0x0A
         };
 
-        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        private static bool CanRunNJulianRuleTest => PlatformDetection.IsNotAndroid && RemoteExecutor.IsSupported;
```

Comment:
Do we know why this test fails on Android?

I would think we would want an open issue tracking enabling this test, right? Can we put a comment to the open issue that tracks enabling it for Android?

Reply:
done opening the issue and adding the comment.